### DISCUSSION TITLE
How does the turbulent wind generated by turbsim superimpose coherent structures?

### DISCUSSION THREAD

**Discussion by 17638031880** [OpenFAST/openfast#2092] [General]
Turbsim has generated turbulent winds, and now we want to add coherent structures between the 100th and 200th seconds. How do we handle this? I'm not very familiar with it

**Comment by jjonkman** [comment]
Dear @17638031880,

The “superimposed coherent turbulent structures” functionality of TurbSim was supported by FAST v7, but is not supported by FAST v8, OpenFAST, or FAST.Farm. 

Best regards,
